modules/control-plane-latency.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/control-plane-latency-recommendations-for-reliable-clusters.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="control-plane-latency_{context}"]
+= Recommended Control Plane latency for reliable clusters
+
+Latency between each of the control plane nodes must be less than 15ms to ensure a well performing and reliable cluster. Some of the metrics to keep track of include etcd gRPC requests latency, fysnc latency and any critical alerts. Here are the PromQL queries:
```

Comment:
Maybe something like. . .

```suggestion
Latency between each of the control plane nodes must be less than `15` ms to ensure an efficient and reliable cluster. Some of the metrics to keep track of include etcd gRPC requests latency, fysnc latency, and any critical alerts.

The following PromQL queries can help monitor latency requirements:
```


Feel free to change "The following PromQL queries can help monitor latency requirements:" to something that might make more sense. I do suggest a stronger lead-in though